Formula/zsh-you-should-use.rb

```diff
@@ -0,0 +1,37 @@
+class ZshYouShouldUse < Formula
+  desc "ZSH plugin that reminds you to use existing aliases for commands you just typed"
+  homepage "https://github.com/MichaelAquilina/zsh-you-should-use"
+  url "https://github.com/MichaelAquilina/zsh-you-should-use/archive/1.7.3.tar.gz"
+  sha256 "db4486cd12974332ec858d446aff9393dae6be430d425a56d7036d2ce4edeb9e"
+  license "GPL-3.0-or-later"
+
+  bottle :unneeded
+
+  depends_on "zsh" => "5.1"
+
+  def install
+    pkgshare.install "you-should-use.plugin.zsh"
+  end
+
+  def caveats
+    <<~EOS
+      To activate You Should Use, add the following to your .zshrc:
+
+        source #{HOMEBREW_PREFIX}/share/zsh-you-should-use/you-should-use.plugin.zsh
+
+      You will also need to force reload of your .zshrc:
+
+        source ~/.zshrc
+
+      Please check homepage for additional settings:
+        https://github.com/MichaelAquilina/zsh-you-should-use
+
+    EOS
```

Comment:
This should be be in the documentation. Only the location of the file is homebrew specific and allowed here.

Reply:
removed and is in the documentation also